appcenter-crashes/android/src/main/java/com/microsoft/appcenter/reactnative/crashes/AppCenterReactNativeCrashesModule.java

```diff
@@ -223,4 +223,14 @@ public void accept(Boolean alwaysSend) {
     public void sendErrorAttachments(ReadableArray attachments, String errorId) {
         WrapperSdkExceptionManager.sendErrorAttachments(errorId, AppCenterReactNativeCrashesUtils.toCustomErrorAttachments(attachments));
     }
+
+    @ReactMethod
+    public void addListener(String eventName) {
+    /* Required for RN built in Event Emitter Calls. */
+    }
+
+    @ReactMethod
+    public void removeListeners(Integer count) {
+    /* Required for RN built in Event Emitter Calls. */
```

Comment:
Formatting.

Reply:
By the way, do we need such changes for iOS implementation too?

Reply:
This issue affects only Android platform. 

Similar issues:
https://github.com/software-mansion/react-native-reanimated/issues/2297
https://github.com/react-navigation/react-navigation/issues/9882
https://github.com/react-native-netinfo/react-native-netinfo/issues/486

Reply:
ok, please add formatting and update changelog